### PR TITLE
Fix 1395

### PR DESCRIPTION
**Pull request by skitt**

### CODE REVIEW

**Review by dfarrell07 — APPROVED** [review]

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -124,6 +124,7 @@ jobs:
             ovn: ovn
           - extra-toggles: dual-stack
           - extra-toggles: ovn
+          - extra-toggles: ovn-ic
```
Comment (sridhargaddam) [inline .github/workflows/testing.yml:127]: We dont run OVN-IC in 0.15 right, so why this change?

File: scripts/shared/lib/clusters_kind [hunk 2]
```diff
@@ -261,11 +257,47 @@ function download_ovnk() {
     )
 }
 
+function prepare_ovn_ic() {
```
Comment (sridhargaddam) [inline scripts/shared/lib/clusters_kind:260]: If this is only to prepare the env' for OVN to build images from a specific commit, its better to rename it as prepare_for_ovn() or prepare_ovn()

**Review by sridhargaddam — CHANGES_REQUESTED** [review]

File: .github/workflows/testing.yml (hunk 1 above)
Comment (skitt) [reply .github/workflows/testing.yml:127]: Good point, I’ll fix https://github.com/submariner-io/shipyard/pull/1395 to drop that.
